resources/zib-ContactPerson.xml

```diff
@@ -0,0 +1,129 @@
+      <alias value="Contactpersoon" />
+      <mapping>
+        <identity value="zib-contactperson-v3.4-2020EN" />
+        <map value="NL-CM:3.1.1" />
+        <comment value="Contact" />
+      </mapping>
+    </element>
+    <element id="RelatedPerson.relationship">
+      <path value="RelatedPerson.relationship" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="RelatedPerson.relationship:role">
+      <path value="RelatedPerson.relationship" />
+      <sliceName value="role" />
+      <short value="Role" />
+      <definition value="Defines the role of the contact in relation to the patient." />
+      <alias value="Rol" />
+      <binding>
+        <strength value="required" />
+        <description value="RolCodelijst" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.3.1.2--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-contactperson-v3.4-2020EN" />
+        <map value="NL-CM:3.1.2" />
+        <comment value="Role" />
+      </mapping>
+    </element>
+    <element id="RelatedPerson.relationship:relationship">
+      <path value="RelatedPerson.relationship" />
+      <sliceName value="relationship" />
+      <short value="Relationship" />
+      <definition value="Defines the contact’s familial relationship to the patient." />
+      <alias value="Relatie" />
+      <binding>
+        <strength value="required" />
+        <description value="RelatieCodelijst" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.3.1.1--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-contactperson-v3.4-2020EN" />
+        <map value="NL-CM:3.1.3" />
+        <comment value="Relationship" />
+      </mapping>
+    </element>
+    <element id="RelatedPerson.name">
+      <path value="RelatedPerson.name" />
+      <short value="NameInformation" />
+      <definition value="Full name of the contact." />
+      <alias value="Naamgegevens" />
+      <type>
+        <code value="HumanName" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/zib-NameInformation" />
+      </type>
+      <mapping>
+        <identity value="zib-contactperson-v3.4-2020EN" />
+        <map value="NL-CM:3.1.4" />
+        <comment value="NameInformation" />
+      </mapping>
+    </element>
+    <element id="RelatedPerson.telecom">
+      <path value="RelatedPerson.telecom" />
+      <short value="ContactInformation" />
+      <definition value="The contact’s telephone number and/or e-mail address." />
+      <alias value="Contactgegevens" />
+      <type>
+        <code value="ContactPoint" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/zib-Contactpoint" />
```

Comment:
zib-contactpoint bestaat niet. Maar dit worden wrs twee profielen. Pieter is hier mee bezig.